### PR TITLE
New hyperparameters

### PR DESCRIPTION
**Pull request by Ecohen1763**
New hyperparameters after running the sweep. Reran analysis files with new VAE data.

### CODE REVIEW

File: 2.train-VAE/2.fetch_hyper_params.ipynb
Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>Please consider adding comments that explain what these loops do</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/#comment-1244319732'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #3.    # layers = ['threelayer']
</pre><p>condiser removing vestigial line </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>Consider writing these to a <code>json</code> or <code>yaml</code> file for access in the training notebook to avoid hardcoding of hyperparameters.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>some comments here explaining what this plot is might be helpful. I struggle with explaining these plots too, so this might be something that we can both talk through.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>Please consider adding comments that explain what these loops do</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/#comment-1244320394'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #3.    # layers = ['threelayer']
</pre><p>condiser removing vestigial line </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>Consider writing these to a <code>json</code> or <code>yaml</code> file for access in the training notebook to avoid hardcoding of hyperparameters.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>some comments here explaining what this plot is might be helpful. I struggle with explaining these plots too, so this might be something that we can both talk through.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: 2.train-VAE/3.beta_vae_training_extract_model_components.ipynb
Comment (MikeLippincott) [inline 2.train-VAE/3.beta_vae_training_extract_model_components.ipynb:9]: <p>Please see comments left in <code>2.train-VAE/2.fetch_hyper_params.ipynb</code> file about hardcoding values and the possible use of <code>yaml</code> or <code>json</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (MikeLippincott) [inline 2.train-VAE/3.beta_vae_training_extract_model_components.ipynb:9]: <p>Using the power of a notebook here is great. I like to treat notebooks like a laboratory notebook. I try to add comments explaining what a figure is, what the interpretation is and maybe why its useful or the significance. ~3 sentences. A small blurb explaining what KL is might help a new reader on this project such as myself.</p><p> </p><p>Also consider double checking the testing vs validation data, it looks like you have a testing subset of data used above. Maybe check with greg if this is validation or testing data. The whole testing vs validation thing gets confusing for me, happy to talk through in person if wanted. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (MikeLippincott) [inline 2.train-VAE/3.beta_vae_training_extract_model_components.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #7.    extracted_col = metadata['ModelID']
</pre><p>nice, dynamic line of code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

File: 2.train-VAE/2.fetch_hyper_params.ipynb
Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>Please consider adding comments that explain what these loops do</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/#comment-1244320882'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #3.    # layers = ['threelayer']
</pre><p>condiser removing vestigial line </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>Consider writing these to a <code>json</code> or <code>yaml</code> file for access in the training notebook to avoid hardcoding of hyperparameters.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (MikeLippincott) [inline 2.train-VAE/2.fetch_hyper_params.ipynb]: <p>some comments here explaining what this plot is might be helpful. I struggle with explaining these plots too, so this might be something that we can both talk through.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: 3.analysis/scripts/nbconverted/3.t_tests.py
Comment (MikeLippincott) [inline 3.analysis/scripts/nbconverted/3.t_tests.py]: One comment, one question here:

Comment: consider formatting code to span multiple lines for ease of reading i.e.

t_test_adult_vs_ped, 
t_test_male_vs_female, 
t_test_adult_male_vs_ped_male,

You might consider talking with Erik about using a pre-commit hook for a code formatted such as [black](https://black.readthedocs.io/en/stable/)

Question:
Why t-tests across multiple groups? Have you considered also doing an ANOVA and post hoc test? There may exist multiple correct answers, consider adding comments as to why a t-test or ANOVA.

**Review by MikeLippincott — APPROVED** [review]
Overall comments: Consider adding more documentation at the cell level explaining what each chunk is doing. Also, consider adding explanations for figures. 

I want to remind you that my comments are just comments and it's ultimately up to you to make changes. Everything looks really good to me and any comments I left are to improve small things. LGTM, feel free to merge after reading/addressing!

File: 2.train-VAE/2.fetch_hyper_params.ipynb
Comment (Ecohen1763) [reply 2.train-VAE/2.fetch_hyper_params.ipynb]: Good catch, also found another line of vestigial code in the same file

Comment (Ecohen1763) [reply 2.train-VAE/2.fetch_hyper_params.ipynb]: Will do

Comment (Ecohen1763) [reply 2.train-VAE/2.fetch_hyper_params.ipynb]: I'm going to hold off on this for now since we are not only interested in the #1 optimal hyperparameter, but also hyperparameter sets with similar val_loss but greater `latent_dim` and `beta` values

File: 3.analysis/scripts/nbconverted/3.t_tests.py
Comment (Ecohen1763) [reply 3.analysis/scripts/nbconverted/3.t_tests.py]: I initially threw together the t-tests across groups since it was quick and adds further analysis. The most important t-tests were adult vs pediatric and male vs female. In the near future I am planning on switching over to ANOVA.

**Pull request merged**
